library/src/main/java/com/jmolsmobile/landscapevideocapture/configuration/PredefinedCaptureConfigurations.java

```diff
@@ -103,4 +103,9 @@ public int getBitrate(CaptureQuality quality) {
         }
 
     }
+
+    public enum FlashOption {
+        HIDE,HIDE_ON, START_OFF, START_ON;
```

Comment:
Wouldn't it be better to call: FlashOption: NO_FLASH, ALWAYS_ON, START_OFF, START_ON

Reply:
Good idea